protocols/imap/pom.xml

```diff
@@ -75,6 +75,11 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
```

Comment:
please import logback and put a logback-test.xml configuration, it's much more helpful